api/src/opentrons/protocol_engine/types.py

```diff
@@ -14,9 +14,8 @@ class EngineStatus(str, Enum):
 
     IDLE = "idle"
     RUNNING = "running"
-    PAUSE_REQUESTED = "pause-requested"
     PAUSED = "paused"
-    STOP_REQUESTED = "stop-requested"
+    STOP_REQUESTED = "stopRequested"
```

Comment:
Oops, unintentional. Note to self: check our internal conventions on string-value enums. I think dash-case might be internally inconsistent

```suggestion
    STOP_REQUESTED = "stop-requested"
```